src/pages/events/rule-events.md

```diff
@@ -0,0 +1,100 @@
+
+All rule events contain the following information:
+
+*  A unique name for the rule event.
+
+*  The code name of the parent event. You must attach a rule event to a specific registered event.
+
+*  One or more rules.
+
+Each rule contains the following:
+
+* A field that is defined in the parent event.
+
+*  An operator, which is represented as a comparison statement between the value supplied in the parent event's payload and the threshold value.
+
+   The operator must be one of the following:
+
+   | Operator      | Description |
+   | -----------   | ----------- |
+   | `greaterThan` | Checks whether the value supplied in the payload of the event is greater than a specified value. Applicable for integer and float data types. |
+   | `lessThan`    | Checks whether the payload value is less than a specified value. Applicable for integer and float data types. |
+   | `equal`       | Checks whether the payload value matches the specified value. For Boolean data types, use `1` to compare to `true` and `0` to compare to `false`. |
+   | `regex`       | A regular expression that checks for matches. The specified value must be compatible with the  [regular expression match](https://www.php.net/manual/en/function.preg-match.php) |
+   | `in`          | Checks whether the payload value is one of multiple specified values. The value must be a comma-separated list. You do not need to provide additional escape characters. |
+
+*  The value to compare against. When you assign the `regex` operator, you must delimit the regular expression value with valid characters, such as forward slashes (/). For example, `/^TV .*/i`, which checks whether the string starts with the string `TV`, ignoring the case of the letters.
+
+You can create rule events within your module's `io_events.xml` file or from the command line.
+
+## Define rule events in `io_events.xml`
+
+The following example creates and registers a rule event named `plugin.magento.catalog.model.resource_model.product.save_low_stock_event`. Its parent is `plugin.magento.catalog.model.resource_model.product.save`. It defines rules that trigger when all of the conditions are true:
+
+*  The value of `qty` is less than 20
+*  The `category_id` is either 3, 4, or 5
+*  The product `name` contains `TV`
+
+These fields are present and declared in the parent event.
+
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="...">
+    <event name="plugin.magento.catalog.model.resource_model.product.save_low_stock_event"
+           parent="plugin.magento.catalog.model.resource_model.product.save"
+    >
```

Comment:
I don't think this is valid XML. I think you need to move up to the end of the previous line.